ProgramHierarchy.java

```diff
@@ -1,18 +1,75 @@
-/*
- * File: ProgramHierarchy.java
- * Name: 
- * Section Leader: 
- * ---------------------------
- * This file is the starter file for the ProgramHierarchy problem.
- */
-
-import acm.graphics.*;
-import acm.program.*;
-import java.awt.*;
-
-public class ProgramHierarchy extends GraphicsProgram {	
-	public void run() {
-		/* You fill this in. */
-	}
-}
-
+/*
+ * File: ProgramHierarchy.java
+ * Name: 
+ * Section Leader: 
+ * ---------------------------
+ * This file is the starter file for the ProgramHierarchy problem.
+ */
+
+import acm.graphics.*;
+import acm.program.*;
+import java.awt.*;
+
+public class ProgramHierarchy extends GraphicsProgram {	
+	
+	private static final int HEIGHT = 50;
+	
+	private static final int WIDTH = 150;
+	
+	public void run() {
+		

```

Comment:
You should breakdown your program into smaller understandable methods. So the run method should have looked like this
 
```
public void run() {
    drawProgramBox();
    drawConsoleLine();
    drawConsoleBox();
    drawGraphicsLine();
    drawGraphicsBox();
    drawDialogLine();
    drawDialogBox();
}
```